cmd/helm/get_dependencies.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"io"
+
+	"github.com/gosuri/uitable"
+	"github.com/spf13/cobra"
+
+	"helm.sh/helm/v3/cmd/helm/require"
+	"helm.sh/helm/v3/internal/completion"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/cli/output"
+)
+
+var getDependenciesHelp = `
+This command list all dependencies for a given release.
+`
+
+type dependencyElement struct {
+	Name       string `json:"name"`
+	Version    string `json:"version,omitempty"`
+	Repository string `json:"repository"`
+	Enabled    bool   `json:"enabled,omitempty"`
```

Comment:
Can you elaborate why this flag should be shown in the output? the definition of a dependency that was installed is “enabled”, so wouldn’t this always return true? If so, then is there any point relaying this information back to the user?

Reply:
Hi 😄 ,

That's a very good question. I read in the [documentation](https://helm.sh/docs/topics/charts/#the-chartyaml-file) that an addiction can be disabled. I don't know in what context this can be done.

Reply:
@bacongobbler do you want that i delete it ?

Reply:
@bacongobbler do you want that i delete it ?

Reply:
small up ? :)

Reply:
Can anyone tell me if my answer is okay or is there something I'm expected to do?

Reply:
Hi @Prims47. I cannot answer that question for you. I am merely pointing out a question on the design. It's up to the author of the pull request (that's you!) to determine if this makes sense from your point of view. It is up to you to hold these opinions, as I noted in the comment:

> Can you elaborate why this flag should be shown in the output?

A good first step would be to use this pull request and test it against a number of charts in the stable repository, and determine if the enabled flag makes sense from a usability standpoint. That way you'd have a better idea whether a user would need this flag or not.

Hope this helps.

side note: I'm spending most of my time on other projects, and I generally don't respond to pings over the weekend. Sorry for the delay on a response.

Reply:
Thanks for the tips. 

Indeed no stable repository uses this notion. So I removed it.

No worries about the deadline. We all have plans on the side, I completely understand. At worst (after several days) I ping on Slack :)

Reply:
@bacongobbler small ping :)